### ISSUE TITLE
Add contents of new line to poem when line has words.

### ISSUE THREAD

**Issue by wakawakathedev** [dewalor/linku#5]
From HKWD. 

I expect the app would append the contents of the input to the end of the poem when clicking "add new line"

---
How it acts now

When the user clicks on "add new line" it erases the contents of the current input box and all text from input is erased.

**Comment by dewalor** [comment]
Right.  There's a UX issue here.  The "New Line" button creates a new blank line and discards the contents of the line immediately above it.  Most people expect the button to save the text first.  This is currently accomplished using the Enter key on our keyboard, which is not obvious at all.  I can make the change.
